pkg/dns/ibm/private/dnssvcs_provider.go

```diff
@@ -0,0 +1,241 @@
+package private
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/networking-go-sdk/dnssvcsv1"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-ingress-operator/pkg/dns"
+	common "github.com/openshift/cluster-ingress-operator/pkg/dns/ibm"
+	dnsclient "github.com/openshift/cluster-ingress-operator/pkg/dns/ibm/private/client"
+	logf "github.com/openshift/cluster-ingress-operator/pkg/log"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+
+	iov1 "github.com/openshift/api/operatoringress/v1"
+)
+
+var (
+	_                       dns.Provider = &Provider{}
+	log                                  = logf.Logger.WithName("dns")
+	defaultDNSSVCSRecordTTL              = int64(120)
+	DNSSVCSApiEndpoint                   = "https://api.dns-svcs.cloud.ibm.com/v1"
+)
```

Comment:
`defaultDNSSVCSRecordTTL` and `DNSSVCSApiEndpoint` can be declared as consts.  Since `DNSSVCSApiEndpoint` isn't used outside this package, it doesn't need to be exported.  Please add godoc!
```suggestion
)

const (
	// defaultDNSSVCSRecordTTL is the default TTL used when a DNS record
	// does not specify a valid TTL.
	defaultDNSSVCSRecordTTL = int64(120)
	// dnsSvcsApiEndpoint is the API endpoint for IBM Cloud DNS Services. 
	dnsSvcsApiEndpoint      = "https://api.dns-svcs.cloud.ibm.com/v1"
)
```
Actually, could you use `dnssvcsv1.DefaultServiceURL` and do away with `DNSSVCSApiEndpoint` entirely?

Reply:
Using DefaultServiceURL.